cmd/add_container.go

```diff
@@ -211,41 +210,35 @@ func generateContainerName(image string) string {
 	return strings.Split(nameAndVersion, ":")[0]
 }
 
-func findPipelinesForWorkflowAction(workflow, action, pipelineName string, allPipelines map[v1alpha1.Type]map[v1alpha1.Action][]v1alpha1.Pipeline) ([]v1alpha1.Pipeline, int, error) {
+func findPipelinesForWorkflowAction(lifecycle, action, pipelineName string, allPipelines map[v1alpha1.Type]map[v1alpha1.Action][]v1alpha1.Pipeline) ([]v1alpha1.Pipeline, int, error) {
```

Comment:
since you've renamed workflow to lifecycle, would you like to update the function name here to `findPipelinesForLifecycleAction()`?

Reply:
I would! 👍